### ISSUE TITLE
Wrong SHA256 sums on GitHub in Repository

### ISSUE THREAD

**Issue by gades** [bitshares/bitshares-ui#3535]
**Describe the bug**
https://github.com/bitshares/bitshares-ui/releases/tag/5.0.220701

**To Reproduce**
1. Download https://github.com/bitshares/bitshares-ui/releases/download/5.0.220701/BitShares_5.0.220701.dmg
2. sha256sum BitShares_5.0.220701.dmg 
36ada6362f28cef31a2cc59fa7a04817a8992a54c6487df2fe47c9de18d6371b  BitShares_5.0.220701.dmg

**Expected behavior**
Automatic SHA 256 is e48e894d57db948921bc4030af8221c379d13e1c7493b00353ba9d5d53c187d2 but I see 36ada6362f28cef31a2cc59fa7a04817a8992a54c6487df2fe47c9de18d6371b

**Comment by sschiessl-bcp** [comment]
Can you please elaborate how you calculated the checksum?

**Comment by gades** [comment]
`wget https://github.com/bitshares/bitshares-ui/releases/download/5.0.220701/BitShares_5.0.220701.dmg
sha256sum BitShares_5.0.220701.dmg`

36ada6362f28cef31a2cc59fa7a04817a8992a54c6487df2fe47c9de18d6371b  BitShares_5.0.220701.dmg

**Comment by abitmore** [comment]
```
$ sha256sum BitShares_5.0.220701.dmg 
36ada6362f28cef31a2cc59fa7a04817a8992a54c6487df2fe47c9de18d6371b  BitShares_5.0.220701.dmg
```

**Comment by gades** [comment]
but here https://github.com/bitshares/bitshares-ui/releases/tag/5.0.220701 sha256 is e48e894d57db948921bc4030af8221c379d13e1c7493b00353ba9d5d53c187d2, why?
<img width="838" alt="Screenshot 2022-08-23 at 2 29 59 PM" src="https://user-images.githubusercontent.com/4200730/186147214-c9136fb6-e196-4dbe-b0e0-ffa73a8275e5.png">

**Comment by abitmore** [comment]
Looks like it is incorrect to use the `hashFiles()` function.

According to the [document](https://github.com/actions/runner/blob/main/docs/adrs/0279-hashFiles-expression-function.md), it is hashed twice:
>Hashing logic:
> 1. Get all files under `$GITHUB_WORKSPACE`.
> 2. Use search pattern filter all files to get files that matches the search pattern. (search pattern only apply to file path not folder path)
> 3. Sort all matched files by full file path in alphabet order.
> 4. Use SHA256 algorithm to hash each matched file and store hash result.
> 5. Use SHA256 to hash all stored files' hash results to get the final 64 chars hash result.
>

**Comment by abitmore** [comment]
I think it will be fixed by #3537.
